include/baresip.h

```diff
@@ -457,6 +457,8 @@ int  message_listen(struct message *message,
 void message_unlisten(struct message *message, message_recv_h *recvh);
 int  message_send(struct ua *ua, const char *peer, const char *msg,
 		  sip_resp_h *resph, void *arg);
+int message_encode_dict(struct odict *od, struct ua *ua, const struct pl *peer,
+			      const struct pl *ctype, struct mbuf *body);
```

Comment:
since only "account" is used inside the function, it might be better to
replace "struct ua *ua" with "struct account *acc" ...?